packages/animations/lib/src/utils/composite_animation_widget.dart

```diff
@@ -0,0 +1,253 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
this widget should actually directly live in the src directory (not under utils) and we should export it from `animations.dart`. I think this class can be handy for other use cases as well.

Reply:
got it